workflow/apm/services.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Services
+sidebarTitle: Services
+---
+
+The Services page displays all of your APM data in a unified view. A service groups together interconnected endpoints, queries, or jobs within your application. This is useful for contextualizing broader issues within your software infrastructure. 
+
+This landing page displays common metrics like requests, latency, and errors. Use the __Resources__, __Traces__, and __Logs__ tabs to drill into your data. This page is available across other APM and log pages to easily understand how data is related to other aspects of your application. Clicking on any of the charts or tables within this page will navigate you to a deeper view of that selected service, including resources, traces, and metrics.
+
+# Resources
+
+Resources are all of the endpoints being requested for a given service, such as CPU, memory, and network bandwidth. 
```

Comment:
I think you missed my comment here - the resources here have nothing to do with CPU, Memory, or network bandwidth. A resourcce here is an endpoint accessible from an application, and the resources section is an aggregate view of how each of those endpoints is performing, showing p50-p99 latency and other performance related statistics.